README.md

```diff
@@ -100,6 +100,8 @@ This project exists thanks to all the [contributors]((https://github.com/globoco
 
 huskyCI is licensed under the [BSD 3-Clause "New" or "Revised" License](https://github.com/globocom/huskyCI/blob/master/LICENSE.md).
 
+*This article can also be read in [Brazilian Portuguese](README-ptBR.md).*
```

Comment:
You can put this line at the top of the document.

Reply:
ok... solved